### DISCUSSION TITLE
"A component is changing an uncontrolled input to be controlled" warning for Input box but not Textarea?

### DISCUSSION THREAD

**Discussion by bassamanator** [jaredpalmer/formik#3381] [Q&A]
I'm using chakra components Input and Textarea. For some reason, I get this error in the console when I type into the Input box and NOT when I type into the Textarea. Validation and submission work however.

I've followed the official [docs here](https://formik.org/docs/api/fastfield). What am I doing wrong?

Thanks.

```
// Input
<FastField name={qid} key={qid} id={qid}>
    {({ field }: { field: any }) => (
	<Input borderColor='gray.200' {...field} />
    )}
</FastField>

// Textarea
<FastField name={qid} key={qid} id={qid}>
    {({ field }: { field: any }) => (
	<Textarea borderColor='gray.200' {...field} />
    )}
</FastField>
```

**Comment by johnrom** [comment]
To remove this warning, pass an initial value to `<Formik initialValues={{ yourField: "" }} />`

**Reply by johnrom** [reply]
It's likely the textarea input simply doesn't display this warning in React, or the library you're using sets it to an empty string by default since a textarea generally only ever reflects a string value. `input`s can reflect a date, time, number, etc and other values that may not be compatible with an empty string per se.

**Comment by bassamanator** [comment]
@johnrom Thanks for the reply. The issue is that this is a dynamic form, there are `n` number of fields, so my `initialValues={{}}`.

I will try to reproduce the pattern you posted on the issue I created.
